README.md

```diff
@@ -229,6 +230,14 @@ recorded values will not be recalculated if this value is changed. If you would
 like to change previous values, you would need to write a migration of the data
 to recalculate history.
 
+## Creating accounts
+
+Follow the login screen's "Sign Up" link, and use the registration key specified
+in the server configuration above.
```

Comment:
Should we mention something here about "if you're a user of conbench, you might need to talk to your conbench administrator to get the registration key"?